src/BitWriter_3_2/BitcodeWriter.cpp

```diff
@@ -0,0 +1,1919 @@
+//===--- Bitcode/Writer/BitcodeWriter.cpp - Bitcode Writer ----------------===//
```

Comment:
@steven-johnson , good point regarding license, thank you. Added LICENSE.TXT for BitWriter component in c329c6a.